RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 -----
 - [***] Products: Cache product images to reduce network requests. [https://github.com/woocommerce/woocommerce-ios/pull/11902]
 - [***] Stats: The Analytics Hub now includes links to the full analytics report for each supported analytics card (Revenue, Orders, Products). [https://github.com/woocommerce/woocommerce-ios/pull/11920]
+- [*] Orders: Show the order attribution info in the order detail screen. [https://github.com/woocommerce/woocommerce-ios/pull/11963]
```

Comment:
To me, this feature looks quite relevant. I think it deserves two stars.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/11963/commits/b4240c8ed5336d3a7d4b024c21b02885e3ce9f9d